spec/system/admin/banners_spec.rb

```diff
@@ -73,8 +73,6 @@
   end
 
   scenario "Publish a banner" do
-    section = create(:web_section, name: "proposals")
```

Comment:
I know this line isn't necessary, but IMHO it makes the test easier to follow. Otherwise there might be a few seconds where the developer is wondering why the section already exists :thinking:.

If you think it's confusing to create a section that already exists, we can also create a section with a different name. What do you think?

Reply:
Good idea! 💡

Reply:
Here i decide to replace the `create` with  `section = WebSection.find_by(name: "proposals")` so latest test expectations keep working without changing anyting else.

Reply:
:ok_hand: